### PR TITLE
Quote file path in soffice command.

### PR DESCRIPTION
**Pull request by jonathandixon**
Fix for #163 

Quote the file path to prevent a bash syntax error.

### CODE REVIEW

File: lib/hydra/derivatives/processors/document.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@ class Document < Processor
     include ShellBasedProcessor
 
     def self.encode(path, format, outdir)
-      execute "#{Hydra::Derivatives.libreoffice_path} --invisible --headless --convert-to #{format} --outdir #{outdir} #{path}"
+      execute "#{Hydra::Derivatives.libreoffice_path} --invisible --headless --convert-to #{format} --outdir #{outdir} \"#{path}\""
```
Comment (jcoyne) [inline lib/hydra/derivatives/processors/document.rb]: Should we use `Shellwords.escape()` here?  This would handle files that include a quote.

Comment (jonathandixon) [inline lib/hydra/derivatives/processors/document.rb]: I wasn't aware of that. It sounds like a good idea. What about all the other places that just quote `#{path}`, should those be updated as well?

Comment (jcoyne) [inline lib/hydra/derivatives/processors/document.rb]: @jonathandixon if you want to fix all the other places, please go ahead. 👍

**Comment by mjgiarlo** [comment]
@jonathandixon @jcoyne Do you think we ought to cut a new release of hydra-derivatives now/soon?

**Comment by jonathandixon** [comment]
I don't have a strong opinion. I pointed my Hyrax app at my fork of hydra-derivatives.

**Pull request merged**